vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/node/ElementNodeConstructorScalarEvaluator.java

```diff
@@ -332,4 +365,65 @@ private void freeENB(ElementNodeBuilder enb) {
     protected boolean createsDictionary() {
         return true;
     }
+
+    private void copyText(TaggedValuePointable tvp, NodeTreePointable ntp, IMutableValueStorage mvs) throws IOException {
```

Comment:
Would it make sense to have all these copy methods in their respective constructor classes?